apps/sharing/tests.py

```diff
@@ -50,9 +50,6 @@ class SharingModelsTestCase(BaseTestCase):
 
     def test_share_count(self):
         addon = Addon.objects.get(id=3615)
-
-        eq_(addon.share_counts[DIGG.shortname], 29)
-
```

Comment:
Indeed, DIGG doesn't exist anymore, but now we are left without a test making sure that the share counts is correct. I think we need to fix that.

Idea: let's modify the `apps/sharing/fixtures/sharing/share_counts.json fixture to replace the obsolete services with the new ones. For example you could replace `digg` with `tumblr`, and then modify the above line to be `eq_(addon.share_counts[TUMBLR.shortname], 29)`.

What do you think?